R/utils-classes.R

```diff
@@ -243,13 +236,18 @@ collect.bcdc_promise <- function(x, ...){
       limit_param = "count",
       offset_param = "startIndex",
       limit = number_of_records,
-      limit_chunk = 5000,
+      limit_chunk = 1000,
       progress = TRUE
     )
 
 
     message("Retrieving data")
-    cc$get(query = query_list)
+
+    tryCatch(cc$post(body = query_list, encode = "form"),
```

Comment:
This shows my ignorance of `crul` and `R6` and might negate my suggestion above for the `post_it` utility function... this call to `cc$post` doesn't assign anything... does that mean `cc` updates itself?

Reply:
Yes `cc` does update itself. Again open to other ways to do this but I think it worthwhile to have this fails on our terms rather than the user getting odd `curl` errors.